apps/librechat/docker-compose.yml

```diff
@@ -0,0 +1,111 @@
+version: "3.7"
+
+services:
+  librechat:
+    container_name: librechat
+    ports:
+      - "${APP_PORT}:3080"
+    depends_on:
+      - mongodb
+      - rag_api
+    image: ghcr.io/danny-avila/librechat-dev:638ac5bba61a524cc4ae99711a91f19572c4f2a0
```

Comment:
version should be fixed like v0.7.4-rc1 instead of hash, also dev version are not prefered